pkg/rpc/client/wsclient.go

```diff
@@ -80,6 +81,8 @@ const (
 
 	// Write deadline.
 	wsWriteLimit = wsPingPeriod / 2
+
+	connCloseErrStr = "use of closed network connection"
```

Comment:
`var connClosedUser = errors.New("connection closed by user")`

The in the `Close()` method:

`setCloseErr(connClosedUser)`

And in the `GetError()`

```
if c.closeErr == connClosedUser {
    return nil
}
```
Will it solve the problem?

Reply:
Yes, nice solution, implemented.